src/libcore/result.rs

```diff
@@ -901,7 +901,12 @@ impl<'a, T> DoubleEndedIterator for IterMut<'a, T> {
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<'a, T> ExactSizeIterator for IterMut<'a, T> {}
 
-/// An iterator over the value in a `Ok` variant of a `Result`.
+/// An iterator over the value in a `Ok` variant of a `Result`. This struct is
+/// created by the `into_iter` method on [`Result`][`Result`] (provided by the
```

Comment:
You don't have to write "[`Result`][`Result`]". Just "[`Result`]" is fine.

Can you also urls for `into_iter` please?